demos/python_demos/colorization/README.md

```diff
@@ -0,0 +1,60 @@
+# Colorization Demo
+
+This example demonstrates an example of using neural networks to colorization video. 
+You can use the following pre-trained models with the demo:
+
+* `colorization-v2`
+* `colorization-v2-norebal`
+
+For more information about the pre-trained models, refer to the [model documentation](../../models/intel/index.md).
```

Comment:
Please recheck links (here and in `See Also` section) with relative paths.

Reply:
@eizamaliev fixed

Reply:
It seems the link is still broken

Reply:
Fixed again))